dashboard/test/lib/contact_rollups_v2_test.rb

```diff
@@ -19,28 +21,41 @@ class ContactRollupsV2Test < ActiveSupport::TestCase
 
   test 'sync new contact' do
     # Create seed data in a source table
-    email = 'test@domain.com'
-    create :email_preference, email: email, opt_in: true
+    email_preference = create :email_preference, email: 'test@domain.com', opt_in: true
+    student_with_parent_email = create :student, parent_email: 'caring@parent.com'
 
     # Stub out Pardot responses
-    pardot_id = 1
     PardotV2.stubs(:retrieve_prospects).
```

Comment:
Since you're here already, could you please add an explanation for these invocation numbers?
* `PardotV2.retrieve_prospects` is expected to run twice because we download new PardotIDs twice in the pipeline.
* `PardotV2.submit_batch_request` is expected to run only once because we have only 2 prospects to create (they can fit 1 in request), and no update. This promise could be even stronger by checking the API endpoint and number of prospects in the request sent. But that could be for later.